docs/resources/Channel.md

```diff
@@ -230,6 +234,7 @@ Represents a message sent in a channel within Discord.
 | CHANNEL_FOLLOW_ADD                     | 12    |
 | GUILD_DISCOVERY_DISQUALIFIED           | 14    |
 | GUILD_DISCOVERY_REQUALIFIED            | 15    |
+| INLINE_REPLY                           | 19    |
```

Comment:
Why is this documented as `INLINE_REPLY`? In the client it seems to just be `REPLY`, which is simpler